.travis.yml

```diff
@@ -48,10 +47,9 @@ before_script:
  - cp composer.json composer.json.old
  - cp composer.json.phpcs composer.json
  - composer self-update
- - composer require squizlabs/php_codesniffer:${PHPCS_VERSION}
+ - composer require squizlabs/php_codesniffer:">=2.0"
```

Comment:
I prefer the "^2.0.0" syntax as it is less prone to breaking things.
See: https://getcomposer.org/doc/articles/versions.md#caret